agent/src/main/scala/za/co/absa/atum/agent/AtumAgent.scala

```diff
@@ -55,6 +55,14 @@ class AtumAgent private[agent] () {
     dispatcher.saveCheckpoint(checkpoint)
   }
 
+  /**
+   * Sends the `Metadata` to the Atumservice API
+   * @param AdditionalData the metadata to be saved to the server.
```

Comment:
parameters in Scala start with lowercase character: https://docs.scala-lang.org/style/naming-conventions.html

Reply:
on few other places as well